assets/components/subscriptionsByCountryGroup/subscriptionsByCountryGroup.jsx

```diff
@@ -5,64 +5,330 @@
 import React from 'react';
 import { connect } from 'react-redux';
 
-import PaperSubscriptionsContainer from 'components/paperSubscriptions/paperSubscriptionsContainer';
-import DigitalSubscriptionsContainer from 'components/digitalSubscriptions/digitalSubscriptionsContainer';
-import InternationalSubscriptions from 'components/internationalSubscriptions/internationalSubscriptionsContainer';
 import { type HeadingSize } from 'components/heading/heading';
+import ThreeSubscriptions from 'components/threeSubscriptions/threeSubscriptions';
+import SubscriptionBundle from 'components/subscriptionBundle/subscriptionBundle';
+import { gridImageProperties } from 'components/threeSubscriptions/helpers/gridImageProperties';
 
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
 import { type CommonState } from 'helpers/page/page';
+import { displayPrice } from 'helpers/subscriptions';
+import { addQueryParamsToURL } from 'helpers/url';
+import { getCampaign, type ReferrerAcquisitionData } from 'helpers/tracking/acquisitions';
+import {
+  getSubsLinks,
+  type SubsUrls,
+  iOSAppUrl,
+  androidAppUrl,
+  dailyEditionUrl,
+} from 'helpers/externalLinks';
+import { classNameWithModifiers } from 'helpers/utilities';
+import { countryGroups } from 'helpers/internationalisation/countryGroup';
+import { type ImageId } from 'helpers/theGrid';
 
 
 // ----- Types and State Mapping ----- //
 
 type PropTypes = {
   countryGroupId: CountryGroupId,
   headingSize: HeadingSize,
+  referrerAcquisitionData: ReferrerAcquisitionData,
 };
 
-
 function mapStateToProps(state: { common: CommonState }) {
 
   return {
     countryGroupId: state.common.internationalisation.countryGroupId,
+    referrerAcquisitionData: state.common.referrerAcquisitionData,
   };
 
 }
 
 
+// ----- Setup ----- //
+
+const appReferrer = 'utm_source=support.theguardian.com&utm_medium=subscribe_landing_page';
+const internationalAppReferrer = 'utm_source=support.theguardian.com&utm_medium=subscribe_landing_page&utm_campaign=international_subs_landing_pages';
```

Comment:
Not sure about using `utm_campaign` to distinguish between UK & international here. Might be better to set the campaign to the country perhaps? That would be useful and would mean use the same form of referrer string for all countries

Reply:
I carried this across from what was there before, it did seem odd to me, but I wanted to keep the functionality the same in this PR because it's quite large.

Reply:
Fair enough, if it's easy for you to change it would be good to but up to you.

Reply:
Updated to use country (technically `supportInternationalisationId`) as campaign.